packages/service-library/src/servicelib/rabbitmq.py

```diff
@@ -21,7 +21,9 @@
 def _connection_close_callback(sender: Any, exc: BaseException | None) -> None:
     if exc:
         if isinstance(exc, asyncio.CancelledError):
-            _logger.info("Rabbit connection was cancelled")
+            _logger.info("Rabbit connection cancelled")
+        elif isinstance(exc, aiormq.exceptions.ConnectionClosed):
+            _logger.info("Rabbit connection closed")
```

Comment:
Please consider using the following else statement. Since there are errors when trying to serialise the sender when being closed under some circumstances.
I have this one in https://github.com/ITISFoundation/osparc-simcore/pull/3941

```python
else:
    _logger.error("Rabbit channel closed with exception from %s", exc)
```